### PR TITLE
Add the pop-up additional BoxShadow property.

### PR DESCRIPTION
**Pull request by wj8400684**
截屏2024-01-07 13 34 44

### CODE REVIEW

File: DialogHost.Avalonia/DialogHost.axaml
Comment (SKProCH) [inline DialogHost.Avalonia/DialogHost.axaml:1]: Can you revert the formatting changes in this file, please?

**Review by SKProCH — CHANGES_REQUESTED** [review]
Everything else looks good

File: DialogHost.Avalonia/DialogHost.axaml
Comment (wj8400684) [reply DialogHost.Avalonia/DialogHost.axaml:1]: That's about it. Please change it.

**Comment by SKProCH** [comment]
Hello, @wj8400684 

Why you closed the PR?

**Pull request closed without merging**